src/views/order_list_view.js

```diff
@@ -0,0 +1,64 @@
+import Backbone from 'backbone';
+import Order from '../models/order';
+import OrderView from '../views/order_view';
+import OrderList from '../collections/order_list';
+import _ from 'underscore';
+
+const OrderListView = Backbone.View.extend({
+  initialize(params){
+    this.template = params.template;
+    this.listenTo(this.model, 'update', this.renderOrders)
+  },
+  events: {
+    'click .btn-buy, .btn-sell': 'addOrder',
+  },
+  renderOrders(){
+
+    this.model.each((order) => {
+
+      const orderView = new OrderView({
+        model: order,
+        template: this.template,
+        tagName: 'li',
+        className: 'order',
+        // bus: this.bus,
+      });
+      this.$('#orders').append(orderView.renderOrder().$el);
+    });
+    return this;
+
+  },
+
+  addOrder(event) {
+    event.preventDefault();
+    const formData = this.getFormData();
+    formData['buy'] = event.target.classList[0] === ('btn-buy')  ? true : false;
+    const newOrder = new Order(formData);
```

Comment:
At this point you need to connect the new `Order` with its corresponding `Quote`. Probably the cleanest way to do this would be to add a function to `QuoteList` to get a `Quote` by symbol, and then pass the `QuoteList` into the `OrderListView` at instantiation.

This will allow you to validate the `Order` based on the current price, and to listen for price changes on the `Quote` so the `Order` can execute automatically when the price is right.